docker-compose/camunda-8.6/docker-compose.yaml

```diff
@@ -385,38 +408,40 @@ services:
       test: /usr/bin/nc -v localhost 1025
       interval: 30s
     networks:
-      - modeler
+      - web-modeler
 
-  modeler-restapi:
+  web-modeler-restapi:
     profiles:
-      - web-modeler
-    container_name: modeler-restapi
-    image: registry.camunda.cloud/web-modeler-ee/modeler-restapi:${CAMUNDA_WEB_MODELER_VERSION}
+      - full
+      - modeling
+    container_name: web-modeler-restapi
+    image: camunda/web-modeler-restapi:${CAMUNDA_WEB_MODELER_VERSION}
```

Comment:
Just for my understanding, is there any reason why each component has its own Docker repo?

Reply:
Three different images are not suitable for the structure of Docker Hub to be placed in just one repository, so we decided to have a repository for each component. See the answer in [this Slack thread](https://camunda.slack.com/archives/C5AHF1D8T/p1715784047909109?thread_ts=1715685967.905449&cid=C5AHF1D8T).